### ISSUE TITLE
Unable to select items from list randomly in accessibility mode

### ISSUE THREAD

**Issue by skrajput3** [nanihadesuka/LazyColumnScrollbar#50]
When I placed my LazyColumn inside LazyColumnScrollbar and enable talkback mode. Items from list are not getting focus. Its works fine if i use only LacyColumn.
